services/ymax-planner/src/engine.ts

```diff
@@ -304,6 +313,7 @@ type Powers = {
   spectrum: SpectrumClient;
   cosmosRest: CosmosRestClient;
   signingSmartWalletKit: SigningSmartWalletKit;
+  now: () => number;
```

Comment:
Suggestion for clarifying that the relevant units are milliseconds:
```suggestion
  now: typeof Date.now,
```